clients/client-direct/src/index.ts

```diff
@@ -290,10 +290,10 @@ export class DirectClient {
 				if (process.env.INSTRUMENTATION_ENABLED === "true") {
 					// Import the runtime instrumentation and instrumentation singleton
 					const instrumentationModule = await import(
-						"@realityspiral/agent/instrumentation"
+						"@realityspiral/plugin-instrumentation"
```

Comment:
We don't need to dynamic import. We can just import it and only use it if needed so feel free to update that.